cmd/keymaster/main.go

```diff
@@ -496,6 +497,10 @@ func main() {
 	flag.Usage = Usage
 	flag.Parse()
 	logger := cmdlogger.New()
+	if *printVersion {
+		fmt.Println(Version)
+		return
+	}
```

Comment:
this does not look clean... but I dont have any good suggestions now on how to make it better.